packages/react-server-cli/package.json

```diff
@@ -48,6 +48,9 @@
   },
   "devDependencies": {
     "babel-preset-react-server": "^0.4.4",
+    "gulp": "^3.9.1",
+    "gulp-babel": "^6.1.2",
+    "gulp-eslint": "^3.0.1",
```

Comment:
Maybe I'm missing the correct dev entry point... but when I cloned and then ran `npm run bootstrap` I hit the same gulp error.